crates/ef-testing/src/models/suite.rs

```diff
@@ -10,7 +10,8 @@ pub struct BlockchainTestSuite {
 }
 
 impl BlockchainTestSuite {
-    pub fn new(name: String) -> Self {
+    #[must_use]
```

Comment:
why adding this? do you think every `new` should have it?

Reply:
good question! on this matter:

```
Clippy pedantic will only ask you to add #[must_use] on functions which

- are publicly visible

- have no (internally) mutable arguments

- don't mutate any statics

- return neither ! nor () nor any #[must_use] type
```

Because there is no way to ensure at compile time that this isn't dead code, but with must_use, you gain protection against dead code